index.html

```diff
@@ -157,43 +169,46 @@ <h2 class="about-title crete">About me</h2>
 
         <ul class="skills">
             <li class="skills-title">
-                <img src="./icons/Rectangle 55.png" alt="" />
-                <h2 class="skills-type inter700">Language</h2>
+                <h2 class="skills-type inter700">
+                    <img src="./icons/Rectangle 55.png" alt="" /> Language
+                </h2>
```

Comment:
- [ ] I'm sorry that you seem confused by the recommendations you were given. The reviewers meant that instead of having your `Skills`, `Language` and `Frameworks` wrapped in a `h2`, it should be in a `li` tag for proper semantic structuring.
- Kindly implement this please... an example will be:
```
<ul>
  <li>Language</li>
    <ul>
      <li>Html</li>
    </ul>
  <li>Framework</li>
     <ul>
      <li>React</li>
    </ul>
</ul>

Reply:
Ok, Its done